config/locales/subscriber_authentication.yml

```diff
@@ -1,18 +1,18 @@
 en:
   subscriber_authentication:
-    heading: Manage your subscriptions
+    heading: Manage your email subscriptions
```

Comment:
@vanitabarrett Is it possible to set different headings for the 2 screens in this journey?

1. Manage your email subscriptions
2. Check your emails

Reply:
@karlchillmaid Yep, that looks possible - I'll take a look!

Reply:
@karlchillmaid I've added a commit with this change 👍